src/Nancy/DefaultRuntimeEnvironmentInformation.cs

```diff
@@ -35,10 +36,20 @@ private static bool GetDebugMode(ITypeCatalog typeCatalog)
             {
                 var assembliesInDebug = typeCatalog
                     .GetTypesAssignableTo<INancyModule>(TypeResolveStrategies.ExcludeNancy)
+#if DOTNET5_4
+                    .Select(x => x.GetTypeInfo().Assembly.GetCustomAttributes(typeof(DebuggableAttribute)))
+                    .Where(x => x.Any());
+
+                    return assembliesInDebug.Any(d => ((DebuggableAttribute)d.First()).IsJITTrackingEnabled);
+#else
                     .Select(x => x.Assembly.GetCustomAttributes(typeof(DebuggableAttribute), true))
                     .Where(x => x.Length != 0);
+                    
+                    return assembliesInDebug.Any(d => ((DebuggableAttribute)d[0]).IsJITTrackingEnabled);

```

Comment:
Not sure if this is the correct way, but using `PlatformServices.Default.Application.Configuration` we can get a string telling us the active configuration at runtime, which by default will be `Debug` when you run a debug build. Perhaps these is a more "solid" approach which would let us know if you run a custom configuration in debug-mode though?